Module4_Concepts/Dijkstrasdirected.md

```diff
@@ -0,0 +1,39 @@
+<!--title={Dijkstra's Algorithm Directed graph}-->
+
+Dijkstra's Algorithm solves one of the more common questions in computer science: the shortest path from location a to location b. Dijskstra's algorithm works specifically on graphs, both directed and undirected. Here is the pseudo code for the algorithm:
```

Comment:
can you flesh this example out a little more?

Reply:
Uh, can you specify what example you want me to specify?

Reply:
from location a to location b. Like how is this rooted in real like? (nissan leaf mileage limitations, gas prices, plane tickets, roadtrip planning, etc)